services/app-api/forms/qm.ts

```diff
@@ -229,5 +260,212 @@ export const qmReportTemplate: ReportTemplate = {
         },
       ],
     },
+    [MeasureTemplateName["LTSS-1"]]: {
+      id: "LTSS-1",
+      title: "LTSS-1: Comprehensive Assessment and Update",
+      type: PageType.Measure,
+      substitutable: true,
+      sidebar: false,
+      elements: [
+        {
+          type: ElementType.ButtonLink,
+          label: "Return to Required Measures Results Dashboard",
+          to: "req-measure-result",
+        } as ButtonLinkTemplate,
```

Comment:
I was getting some insane type errors that made me cast every single element like this or else tests would fail to run. I have no clue how to resolve this otherwise but would love some help with this since casting like this is untenable.

Reply:
I removed all the elements but just a warning this might come up later and hopefully someone who's better at typescript has to deal with it instead of me